### PR TITLE
Adds info about proctoring

### PR DESCRIPTION
**Pull request by lamagnifica**
@smagoun @nedbat

### CODE REVIEW

**Comment by openedx-webhooks** [comment]
## [DOC-2789](https://openedx.atlassian.net/browse/DOC-2789)

Adds more info about proctoring in Dogwood.

### Reviewers
- [x] Subject matter expert: @smagoun 
- [x] Doc team review (sanity check): @catong @pdesjardins @srpearce 

FYI: @nedbat 

### Testing
- [ ] Ran ./run_tests.sh without warnings or errors

### Post-review
- [ ] Add description to release notes task as a comment
- [x] Squash commits

File: en_us/open_edx_release_notes/source/dogwood.rst [hunk 1]
```diff
@@ -175,8 +175,10 @@ the *Building and Running an Open edX Course* guide.
 
 .. note::
    Another special exam feature, proctoring, is available for courses on the
-   edx.org website. The proctoring feature is not supported for Open edX
-   installations at this time.
+   edx.org website. The proctoring feature can be enabled in Open edX, but it
+   is not supported for the Dogwood release. This feature requires additional
+   configuration and the services of a third-party proctoring vendor such as
+   Software Secure to be used effectively.
```
Comment (catong) [inline en_us/open_edx_release_notes/source/dogwood.rst:181]: If we wanted to omit mentioning a specific vendor (by striking "such as Software Secure") the sentence would work just as well.

**Comment by catong** [comment]
One question about whether we want to mention a specific proctoring vendor, otherwise :+1:

**Comment by lamagnifica** [comment]
Sagreed, @catong that makes me uncomfortable as well. @smagoun is the vendor shout out necessary?

**Comment by smagoun** [comment]
@lamagnifica I would prefer to keep the reference as part of our business relationship.

**Pull request merged**